pull_request_template.md

```diff
@@ -4,9 +4,18 @@ Please explain the changes you made. Add pictures or videos to explain them bett
 
 ## Checklist
 
-If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.
+If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **AT LEAST ONE** box from each section has been checked **BEFORE** the PR is merged.
 
+## Tests
+- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure)
 - [ ] Manually tested that added code works as intended (if any functional/runnable code is added).
 - [ ] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
-- [ ] Tested on real hardware (if the changed or added code is used with the real hardware).
+## Tested on
```

Comment:
I'd say this section is redundant if we just add "tested on hardware" in the section above, but not a strong opinion.

Reply:
You have a point here, I am considering removing "Manually tested that added code works as intended (if any functional/runnable code is added)." to just have whether tests where added or not, just so the reviewer is aware to check the tests before approving the PR, what do you think?

Reply:
Personally, I'd rather remove the whole "tested on" section and just add "manually tested (1. in simulation, 2. on hardware)" in the first section, but open for other ideas.

Reply:
Thanks! Updated, let me know what you think

Reply:
@BenStarmerSmith Added a few tiny suggestions